plugins/inputs/modbus/modbus.go

```diff
@@ -88,6 +88,10 @@ const sampleConfig = `
 
   # TCP - connect via Modbus/TCP
   controller = "tcp://localhost:502"
+  
+  # RTU over TCP - connect via Modbus RTU over TCP
+  # controller = "tcp://localhost:502"
+  # transmission_mode = "RTU"
```

Comment:
I think we should either place this further down and fuse in the serial `transmission_mode` setting or make more clear that this is the setting for TCP, e.g. by moving the setting closer to the `controller` setting and place the comment above `controller`.
Please also add a comment like
```toml
  ## For Modbus over TCP you can choose between "TCP", "RTUoverTCP" and "ASCIIoverTCP".
```
and make "TCP" the default under the TCP `controller` setting.

Reply:
I've added a more descriptive message for the different `transmission_mode`s. Transmission mode has been condensed to only one description in the README.